code/modules/recycling/sortingmachinery.dm

```diff
@@ -48,8 +58,45 @@
 			. += note.examine(user)
 	if(sticker)
 		. += "There's a barcode attached to the side."
+	if(sort_tag)
+		. += "There's a sorting tag with the destination set to [GLOB.TAGGERLOCATIONS[sort_tag]]."
+
+/obj/item/delivery/proc/disposal_handling(disposal_source, obj/structure/disposalholder/disposal_holder, obj/machinery/disposal/disposal_machine, hasmob)
+	SIGNAL_HANDLER
+	if(!hasmob)
+		disposal_holder.destinationTag = sort_tag
 
-/obj/structure/big_delivery/attackby(obj/item/W, mob/user, params)
+/obj/item/delivery/relay_container_resist_act(mob/living/user, obj/O)
+	if(ismovable(loc))
+		var/atom/movable/AM = loc //can't unwrap the wrapped container if it's inside something.
```

Comment:
dont use abbreviated names like AM its against our contributing guidelines. use something like movable_loc

Reply:
Like the other one, that was a pre-existing line, but I can change it.

Reply:
All instances of those problems fixed as far as I can see.